README.md

```diff
@@ -50,18 +50,18 @@ git submodule update --checkout --init --recursive test/unit-test/CMock
 
 ## Migration Guide
 
-### How to migrate from v2.0.0 (Release Candidate) to v3.3.0
+### How to migrate from v2.0.0 (Release Candidate) to v3.4.0
```

Comment:
Is v2.0.0 still release candidate?

Reply:
I'm honestly not sure? I feel like it shouldn't be the RC given how far off that tag is. Do we just want to remove that part of it?

Reply:
I think so. I do not think that it is a release candidate now. @gitwallit can you confirm?

Reply:
I believe v2.0.0 was the original release candidate version of the OTA library. Post that, we then did a GA release with v3.0.0. So v2.0.0 is still, and will aways be, the original RC version.